src/components/PaymentConfirmed.vue

```diff
@@ -1,43 +1,94 @@
 <script lang="ts">
+import { formatTime } from '@/utils/utils'
+
 export default {
   name: 'PaymentConfirmed',
   props: {
     paymentAmount: Number,
     referenceId: String,
-    timeStamp: String
+    timeStamp: String,
+  },
+  components: {},
+  computed: {
+    amountCents() {
+      return this.paymentAmount;
+    },
+    currency() {
+      return 'R'; //TODO: get currency from store
+    },
+    paidAt() {
+      return this.timeStamp; 
+    }
   },
-  components: {}
+  methods: {
+    formatTime,
+  }
 }
 </script>
 
 <template>
-  <div class="text-center bg-amber-400 text-black">
-    <span>Payment Complete!</span>
-  </div>
-  <div class="container mx-auto text-center">
+  <div class="container mx-auto text-center payment-success-card">
     <div class="columns-auto">
-      <h4 class="text-gray-200 font-bold py-4">Payment Successfull</h4>
+      <h4 class="text-gray-200 font-bold">Payment Successful</h4>
     </div>
     <div class="columns-auto">
-      <img class="payment-qr-code mx-auto py-5" src="../assets/payment-success.png" />
+      <img class="payment-qr-code mx-auto" src="../assets/payment-success.png" />
     </div>
     <div class="columns-auto">
-      <h1 class="text-gray-200 font-bold py-4">R200</h1>
-      <h4 class="text-gray-200 font-bold py-2">12:05, 13 March 2024</h4>
-      <h4 class="text-green-600 font-bold py-4">Reference ID</h4>
-    </div>
-    <div class="columns-auto py-4 mx-4">
-      <button
-        class="bg-yellow-500 hover:bg-yellow-600 text-black font-bold py-2 px-4 rounded w-full max-w-90"
-      >
-        Close Screen
-      </button>
+      <h1 class="payment-amount">{{currency}} {{paymentAmount}}</h1>
+      <h4 class="time-stamp">{{formatTime(timeStamp)}}</h4>
+      <h4 class="reference-id">{{ referenceId }}</h4>
     </div>
   </div>
+  <div class="columns-auto py-4 mx-4">
+    <button
+      class="close-screen-btn py-2 px-4 rounded w-1/4"
+    >
+      Close Screen
+    </button>
+  </div>
 </template>
 
 <style scoped>
 .payment-qr-code {
   max-width: 300px;
 }
+
+.close-screen-btn {
+  background-color: var(--color-amber-med);
+  font-weight: bold;
+  color: var(--color-black);
+  text-align: center;
+}
+
+button:hover {
+  background-color: var(--color-amber-light);
+}
+
+.payment-success-card {
```

Comment:
I suspect the purple border was just an artifact of what was selected in his editor when he took the screenshot.

it looks kinda wierd to me, I think we can remove it.